addons/stock/tests/test_move.py

```diff
@@ -3559,12 +3559,16 @@ def test_scrap_2(self):
         """ Check the created stock move and the impact on quants when we scrap a
         consumable product.
         """
+        sequence = self.env['ir.sequence'].search([('code', '=', 'stock.scrap'), ('company_id', '=', self.env.user.company_id.id)])
+        number = sequence.number_next_actual
```

Comment:
variable not used

Reply:
actually, I was checking that if it works fine then will optimize the code but you caught me first. my mistake. although I hope this will work?